blocks/library/image/index.js

```diff
@@ -82,40 +85,50 @@ registerBlockType( 'core/image', {
 
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, alt, caption } = attributes;
+		const updateAlt = ( newAlt ) => setAttributes( { alt: newAlt } );
+
+		const inspector = focus && (
+			<InspectorControls key="inspector">
+				<TextControl label={ __( 'Alternate Text' ) } value={ alt } onChange={ updateAlt } />
+			</InspectorControls>
+		);
 
 		if ( ! url ) {
 			const uploadButtonProps = { isLarge: true };
 			const setMediaURL = ( media ) => setAttributes( { url: media.url } );
-			return (
+			return [
+				inspector,
```

Comment:
Aha! Interesting. 

Probably not — we need to think about what shows on the frontend. And if you actually publish an image placeholder (forget to fill it out), I'm assuming you'll see nothing. But if we've offered you to fill out the alt text of a placeholder, feels like there's an implicit expectation that you should see _something_. So I do agree, probably we shouldn't show anything but the basic alignments for placeholders.